processing/processing.py

```diff
@@ -0,0 +1,15 @@
+
+
+def addition(summand1: float, summand2: float) -> float:
```

Comment:
```suggestion
def addition(summand1: float, summand2: float) ->  Dict[str, float]:
```

Der typehint passt nicht ganz, vielleicht so wie oben? (dann fehlt noch ein "from typing import Dict")